app/src/main/java/com/cornellappdev/android/eatery/CampusMenuActivity.java

```diff
@@ -207,6 +226,109 @@ else if (mCafeData instanceof DiningHallModel) {
+        XAxis xAxis = b.getXAxis();
+        xAxis.setDrawGridLines(false);
+        xAxis.setPosition(XAxis.XAxisPosition.BOTTOM);
+
+        // Set X axis colors and fonts.
+        // "xAxis.set[_]Color(R.color.[_])" will use Android's default colors, rather than our custom defined.
+        xAxis.setAxisLineColor(getResources().getColor(R.color.inactive));
+        xAxis.setTextColor(getResources().getColor(R.color.secondary));
+        xAxis.setTextSize(14f);
+        xAxis.setAxisLineWidth(2f);
+        // Setup X axis labels.
+        xAxis.setAxisMaximum(21);
+        xAxis.setLabelCount(8);
+        xAxis.setTypeface(Typeface.SANS_SERIF);
+        xAxis.setYOffset(8f);
+        ArrayList<String> xAxisLabels = new ArrayList<String>();
+        for(int i = 0; i < 22; i++) {
+            int hourNum = (i + 6) % 12;
+            if (hourNum == 0) {
+                hourNum = 12;
+            }
+            String ap = i <= 5 || i >= 18 ? "a" : "p";
+            xAxisLabels.add(hourNum + ap);
+        }
+        xAxis.setValueFormatter(new IndexAxisValueFormatter(xAxisLabels));
+    }
+
+    private void setupWaitTimesChart() {
+        mWaitTimesChart = (BarChart) findViewById(R.id.wait_time_chart);
+        mWaitTimesChart.setDescription(null);
+        mWaitTimesChart.getLegend().setEnabled(false);
+        mWaitTimesChart.setDoubleTapToZoomEnabled(false);
+        mWaitTimesChart.setPinchZoom(false);
+        mWaitTimesChart.animateY(2000);
+        mWaitTimesChart.setExtraBottomOffset(24f);
+        mWaitTimesChart.getRendererXAxis().getPaintAxisLabels().setTextAlign(Paint.Align.LEFT);
+
+
+        mWaitTimesChart.setOnChartValueSelectedListener(new OnChartValueSelectedListener() {
+            @Override
+            public void onValueSelected(Entry e, Highlight h) {
+
+            }
+
+            @Override
+            public void onNothingSelected() {
+
+            }
+        });
+
+        setupAxis(mWaitTimesChart);
+
+        List<BarEntry> entries = new ArrayList<>();
+        entries.add(new BarEntry(0, 2f));
+        entries.add(new BarEntry(1f, 30f));
+        entries.add(new BarEntry(2f, 10f));
+        entries.add(new BarEntry(3f, 20f));
+        entries.add(new BarEntry(4f, 30f));
+        entries.add(new BarEntry(5f, 40f));
+        entries.add(new BarEntry(6f, 50f));
+        entries.add(new BarEntry(7f, 35f));
+        entries.add(new BarEntry(8f, 50f));
+        entries.add(new BarEntry(9f, 30f));
+        entries.add(new BarEntry(10f, 2f));
+        entries.add(new BarEntry(11f, 5f));
+        entries.add(new BarEntry(12f, 10f));
+        entries.add(new BarEntry(13f, 60f));
+        entries.add(new BarEntry(14f, 40f));
+        entries.add(new BarEntry(15f, 20f));
+        entries.add(new BarEntry(16f, 2f));
+        entries.add(new BarEntry(17f, 2f));
+        entries.add(new BarEntry(18f, 10f));
+        entries.add(new BarEntry(19f, 2f));
+        entries.add(new BarEntry(20f, 2f));
```

Comment:
please use a loop

Reply:
oh wait this is temporary data, i get it now. 

please add a comment saying that this is temporary, and add something like `TODO (yanlam): pull wait times from backend and make bar entries dynamic`